decode_test.go

```diff
@@ -19,6 +19,11 @@ const (
 	// Stored at https://gist.github.com/AlekSi/d4369aa13cf1fc5ddfac3e91b67b2f7b
 	London    = "greater-london-140324.osm.pbf"
 	LondonURL = "https://gist.githubusercontent.com/AlekSi/d4369aa13cf1fc5ddfac3e91b67b2f7b/raw/f87959d6c9466547d9759971e071a15049b67ae2/greater-london-140324.osm.pbf"
+
+	// Same file as above, but without 'DenseNodes'. This has been generated using the below command (using osmium-tool http://osmcode.org/osmium-tool/)
+	// "osmium cat -o  greater-london-140324-nondense.osm.pbf greater-london-140324.osm.pbf -f osm.pbf,pbf_dense_nodes=false"
+	LondonNonDense    = "greater-london-140324-nondense.osm.pbf"
+	LondonNonDenseURL = "https://gist.githubusercontent.com/clydedacruz/6263693f4facc28c84e768e708a489ff/raw/0f2ecad796a0234678c8fa6a0cc8fae376ead646/greater-london-140324-nondense.osm.pbf"
```

Comment:
I just added it to the same gist as dense version: https://gist.github.com/AlekSi/d4369aa13cf1fc5ddfac3e91b67b2f7b. Please update that URL.

Reply:
Sure

Reply:
Should I remove the TBD item from the readme as well ?

Reply:
I've updated the gist link